### DISCUSSION TITLE
Welcome to fulcro-exercises Discussions!

### DISCUSSION THREAD

**Discussion by holyjak** [fulcro-community/fulcro-exercises#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by muhajirdev** [comment]
Hi @holyjak thanks for creating this repo. This is very helpful

**Reply by holyjak** [reply]
Thank you! Encouraging feedback is very helpful, as it keeps me going :)
